tests/test_jointcal_decam.py

```diff
@@ -45,15 +45,17 @@ def setUp(self):
         radius = 3*lsst.afw.geom.degrees
         self._prep_reference_loader(center, radius)
 
-        self.input_dir = os.path.join(data_dir, 'data')
+        self.input_dir = os.path.join(data_dir, 'decam')
         self.all_visits = [176837, 176846]
+        ccdnums = '^'.join(str(x) for x in range(10,19))
+        self.other_args = ['ccdnum='+ccdnums,]
```

Comment:
Whitespace after the comma (E231) and also around the + operator (according to the style guide).